kolibri/core/assets/src/utils/appCapabilities.js

```diff
@@ -0,0 +1,7 @@
+import plugin_data from 'plugin_data';
+
+const appCapabilities = plugin_data.appCapabilities || {};
+
+export default {
+  launchIntent: appCapabilities.launch_intent,
+};
```

Comment:
Mostly because there wasn't a clear need for it to be pluggable, so it seemed to make it more complicated.

In this case because we are exporting an object, attempting to check an undefined capability would return a falsy value.

Reply:
Ok, yeah, as long as it's got "graceful degradation" then that seems fine.